__tests__/integration/welcome/createWallets.test.tsx

```diff
@@ -0,0 +1,177 @@
+import puppeteer from "puppeteer";
+import { getDocument, queries, wait } from "pptr-testing-library";
+const { getByText } = queries;
+import { USER } from "complete-randomer";
+
+const delay = async (time) => {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, time);
+  });
+};
+
+const loadExtension = async () => {
+  const extensionID = "lmcohpmphdnfjpoejmnpeffoffngmkoi";
+  const extensionPath = "./dist/development/chrome";
+  const extensionOptionHtml = "welcome.html";
+  const extPage = `chrome-extension://${extensionID}/${extensionOptionHtml}`;
+  const browser = await puppeteer.launch({
+    headless: false,
+    args: [
+      `--disable-extensions-except=${extensionPath}`,
+      `--load-extension=${extensionPath}`,
+      "--no-sandbox",
+      "--disable-setuid-sandbox",
+      "--start-maximized",
+    ],
+  });
+
+  // trick to bring the new welcome page to the front
+  await delay(1000);
+  const page = await browser.newPage();
+  await page.bringToFront();
+  await page.setViewport({ width: 1366, height: 768 });
+  await page.goto(extPage);
+
+  return { page, browser };
+};
+
+describe("Create or connect wallets", () => {
+  it("successfully creates an Alby wallet", async () => {
+    try {
+      const user = USER.SINGLE();
+      const { page, browser } = await loadExtension();
+
+      // go through welcome page
+      const $document = await getDocument(page);
+
+      await wait(() => getByText($document, "Get Started"));
+
+      // go through welcome page
+      await getByText($document, "Get Started").then((button) =>
+        button.click()
+      );
+
+      // type user password and confirm password
+      await page.focus("#password");
+      await page.keyboard.type(user.password);
+
+      await page.focus("#passwordConfirmation");
+      await page.keyboard.type(user.password);
+
+      // submit password form
+      await getByText($document, "Next").then((button) => button.click());
+
+      // click at "Create Alby Wallet"
+      await getByText($document, "Create a new wallet").then((button) =>
+        button.click()
+      );
```

Comment:
nit: why not this (same applies for all other "then" calls)?

```suggestion
      const createNewWalletButton = await getByText($document, "Create a new wallet");
      
      createNewWalletButton.click();
```

Reply:
Nice feedback! 
I'll push a commit to normalize all the await calls! 

Thanks, bro! :heart: